### ISSUE TITLE
Collapsible 'show' elements failing WCAG 4.1.1 Parsing (Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#1996]
A recent DAC audit has flagged a level A issue with the collapsible component. 

WCAG Reference: 
4.1.1 Parsing (Level A)

The ‘Show’ elements are not nested accordingly to the HTML5 standards. The ‘Show’ 
elements are buttons that are nested within a link, and this is not valid markup. This may 
cause problems for assistive technology. 

```
<div class="ons-collapsible__heading ons-js-collapsible-heading svelte-1gosn14" 
role="link" aria-controls="topic-0" aria-expanded="false" data-ga-action="Close 
panel">
[...] 
<button type="button" class="ons-btn ons-collapsible__btn ons-js-collapsible-
button ons-u-d-no@xxs@s ons-btn--secondary ons-btn--small" aria-controls="topic-0" 
data-ga-action="Close panel"> 
<span class="ons-btn__inner ons-js-collapsible-button-inner">Show</span> 
</button> 
[...]
</div> 
```

_Originally posted by @James-Nitron in https://github.com/ONSdigital/design-system/discussions/148#discussioncomment-2206300_

**Comment by boxadesign** [comment]
This is on hold pending testing from DAC. I have created a new version that will need to be tested before going forward.

https://deploy-preview-2013--ons-design-system.netlify.app/components/accordion/
https://deploy-preview-2013--ons-design-system.netlify.app/components/collapsible/
